### ISSUE TITLE
enable appveyor for the main repo

### ISSUE THREAD

**Issue by shawnlaffan** [timbunce/devel-nytprof#208]
This is a follow up to https://github.com/timbunce/devel-nytprof/pull/207#issuecomment-1457209471

The appveyor builds are currently set up to run on the fork at https://github.com/jkeenan/devel-nytprof/

Ideally they will also be on the main repo.

**Comment by toddr** [comment]
windows builds are also available via github actions now.

**Comment by shawnlaffan** [comment]
Yes, but the appveyor config calls Strawberry perl so still has some uses.  

That said, Strawberry perl can also be added to github CI using shogo82148/actions-setup-perl.  
https://github.com/shogo82148/actions-setup-perl#use-strawberry-perl-on-windows
